cypress/locators/select/locators.js

```diff
@@ -8,3 +8,4 @@ export const SELECT_LIST = '[data-element="select-list"]';
 export const SELECT_BASIC = '[id="simple"]';
 export const OPEN_ON_FOCUS_ID = '[id="openOnFocus"]';
 export const IS_LOADING_ID = '[id="isLoading"]';
+export const SELECT_INPUT = '[data-element="input"]';
```

Comment:
we don't need to make this as a separate locator, we can use `getDataElementByValue()` function